### PR TITLE
Master add snapshot tests

### PR DESCRIPTION
**Pull request by jkonecny12**
Add pre and post installation snapshots!

### CODE REVIEW

File: snapshot-post.ks.in [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+if [ "$ORIG_UUID" == "$SNAP_UUID" ]; then
+    echo "*** snapshot and origin LV UUID must differ" >> /root/RESULT
+fi
+
+# If we made it post, that's good enough
+if [ ! -e /root/RESULT ]; then
+    echo SUCCESS > /root/RESULT
+fi
+
+%end
```
Comment (poncovka) [inline snapshot-post.ks.in]: A missing new line at the end of file.

File: snapshot-pre.ks.in [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+if [ "$ORIG_UUID" == "$SNAP_UUID" ]; then
+    echo "*** snapshot and origin LV UUID must differ" >> /root/RESULT
+fi
+
+# If we made it post, that's good enough
+if [ ! -e /root/RESULT ]; then
+    echo SUCCESS > /root/RESULT
+fi
+
+%end
```
Comment (poncovka) [inline snapshot-pre.ks.in]: A missing new line at the end of file.

File: snapshot-post.ks.in (hunk 1 above)
Comment (jkonecny12) [reply snapshot-post.ks.in]: Fixed.

File: snapshot-pre.ks.in (hunk 2 above)
Comment (jkonecny12) [reply snapshot-pre.ks.in]: Fixed.

**Comment by jkonecny12** [comment]
*Updated:*
* Add missing newlines

**Pull request merged**